src/storage.cpp

```diff
@@ -0,0 +1,129 @@
+// Include custom storage API
+#include "storage.h"
+// Include custom debug macros and compile flags
+#include "flags.h"
+
+// This storage API is unencrpyted.
+// NOTE: NVS is not directly compatible with the ESP32 flash encryption system.
+//       However, data can still be stored in encrypted form if NVS encryption
+//       is used together with ESP32 flash encryption.
+//       Please refer to NVS Encryption for more details.
+//       NVS Encryption: https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/storage/nvs_encryption.html
+
+// NOTE: May want to add lock around API if it's used across threads,
+//       not necessary right now.
```

Comment:
While `storage_init` is used in more than one thread, like it is right now, this will be necessary.

Reply:
I've removed `storage_init` from `setup`, making it so it's only called in one place for now.